package.json

```diff
@@ -31,7 +31,7 @@
     "json-merge-patch": "^1.0.2",
     "json-pointer": "^0.6.2",
     "json-schema-traverse": "^0.4.1",
-    "jsonpath-plus": "^10.0.0",
+    "jsonpath-plus": "^10.2.0",
```

Comment:
Hopefully tests are still g2g after this update, but this one gets rid of a `critical` issue.

Reply:
Tests pass keeping it.